### PR TITLE
Add test case for getEventTarget

### PR DESCRIPTION
**Pull request by lidoravitan**
Added Test case for packages/react-dom/src/events/getEventTarget.js

### CODE REVIEW

**Comment by FancyCoder0** [comment]
Dear lidoravitan,

We are researchers working on identifying redundant development and duplicated pull requests. We have found there is a pull request: https://github.com/facebook/react/pull/11528 which might be a potentially duplicate to this one. We would really appreciate if you could help us to validate and give us feedback.

Thank you very much for your time!

Sincerely, Luyao

**Comment by lidoravitan** [comment]
@gaearon Thanks for response! Sure I will change the PR by your comment to cover the corner case. and update soon :)

**Comment by lidoravitan** [comment]
@gaearon 
I hope we can promote this PR soon, because I would be happy to contribute more below :)
thanks a lot!

**Comment by lidoravitan** [comment]
@gaearon, please can you check this PR ? thanks.

File: packages/react-dom/src/events/__tests__/getEventTarget-test.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          cancelable: true,
+        });
+
+        expect(target).toEqual(null);
+        container.firstChild.dispatchEvent(nativeEvent);
+        expect(target).toEqual(container.firstChild);
+      });
+
+      // Normalize SVG <use> element events #4963
+      it('get target element object in case the target has correspondingUseElement property', () => {
```
Comment (gaearon) [inline packages/react-dom/src/events/__tests__/getEventTarget-test.js]: If I disable `correspondingUseElement` logic in `getEventTarget.js` this test still passes. Seems wrong.

Comment (lidoravitan) [reply packages/react-dom/src/events/__tests__/getEventTarget-test.js]: @gaearon I agree that it seems wrong, so i fixed that. please review if now it better.
thanks :)

Comment (lidoravitan) [reply packages/react-dom/src/events/__tests__/getEventTarget-test.js]: @gaearon hope you can review this PR again, thanks

**Comment by lidoravitan** [comment]
@gaearon can you review please?

**Comment by lidoravitan** [comment]
@gaearon  @sophiebits Please can you take a look, thanks!

**Comment by lidoravitan** [comment]
@gaearon is it still relevant?

**Comment by lidoravitan** [comment]
@gaearon is it could be relevant? or close this PR? btw, I fixed this PR by your comment.

**Comment by gaearon** [comment]
Hey, thanks for PR. Looking at it again, I'm not sure it adds anything in practice that isn't already covered by other tests. So we probably don't need it after all. Sorry for the churn. The SVG case is interesting, but it's clear enough from the source code itself so if we ever decide to remove this special case, the test won't help us much.

**Pull request closed without merging**